app/helpers/submissions_helper.rb

```diff
@@ -51,6 +51,14 @@ def submission_solitary_author(submission)
     end.join(", ").presence || content_tag(:em, 'unknown author')
   end
 
+  def submission_author_forenames(submission)
+    submission.students.map(&:forename).join(", ").presence || content_tag(:em, 'unknown author')
+  end
+
+  def submission_author_surname(submission)
```

Comment:
This method's name should be pluralized.

```suggestion
  def submission_author_surnames(submission)
```